src/utils/shared.ts

```diff
@@ -18,14 +18,13 @@ export const toHtmlSafeId = (
 };
 
 export const validateSearchField = (value?: string) => {
-  return value ? /^([a-zA-Z0-9-_%]*[a-zA-Z0-9-_%])?$/.test(value.trim()) : true;
+  return value ? /^([a-zA-Z0-9-_ ]*)$/.test(value.trim()) : true;
```

Comment:
maybe I'm having issues with finding the newest commit but I still think we are missing a `.` here (I promise that's my last issue :D)

Reply:
I missed it because of bad `squash` command. I hope it is there now :) https://github.com/bf2fc6cc711aee1a0c2a/cos-ui/blob/3e31595541a564c013d9b3cf4e16ffb2b911f410/src/utils/shared.ts#L20-L26